dist/utils.js

```diff
@@ -389,3 +375,27 @@ export async function waitForResolvedLinks(app, delay = 1000, max = 50) {
         throw Error("Obsidian-Community-Lib: ResolvedLinks did not finish initialising. `max` iterations was reached first.");
     }
 }
+/**
+ * Check if the content of a note has YAML. If so, return an array of the YAML and the rest of the note. If not, return `['', content]`
+ * @param  {string} content
+ */
+export function splitAtYaml(content) {
+    if (!content.startsWith("---"))
```

Comment:
This should work, but you could also use a RegEx and check for something like this: `^---$` (`^` is start of line, `$` is end of line)

Reply:
![image](https://user-images.githubusercontent.com/59741989/146669475-cc299bdd-41b8-4606-8873-fe2b19ddd3e6.png)
Currently, this would be handled like the Frontmatter.

Reply:
Good point, I'll use the regex instead